### DISCUSSION TITLE
ffmedia Environment build error

### DISCUSSION THREAD

**Discussion by jasonfint** [unosquare/ffmediaelement#629] [Q&A]
Error
Unable to Load FFmpeg Libraries from path:c:\ffmpeg\x64
Make sure the above folder contains FFmpeg shared binaries(dll files) for the applicantion's architecture (64-bit)
TIP: You can download builds from
https://ffmpeg.org/download.html
FileNotFoundException: Unable to load FFmpeg binaries fromfolder "c:\ffmpeglx64.
Application will exit.

I have created the C:\ffmpeg\x64 folder on the c drive, and put the downloaded ffmpeg.exe, ffplay.exe, and ffprobe.exe in this folder. But I don't see any other dll files in the release package downloaded from the official website? How to solve this problem?

**Comment by ghost** [comment]
@jasonfint Hi, You need to go to BtbN Git : https://github.com/BtbN/FFmpeg-Builds/releases

Find ffmpeg version and download the "win64-gpl-shared"
Now you'll have all the dll files that you can copy with the exe in to your folder !
